### PR TITLE
Update microsoft-365-groups-expiration-policy.md

### PR DESCRIPTION
**Pull request by simonagren**
- When deleting a group or team, most of the associated resources are also deleted. Exceptions include:
Videos in Stream remain and are owned by the person who uploaded/recorded them
Flows in Power Automate remain and are owned by the person who created them.
Project and roadmap data in Project on the web remains in the CDS and can be restored separately.

- For me the usage of group owners vs team owners where a bit unclear. A team owner is a group owner as well

- Conflicting information about Yammer renewal

- Microsoft Purview instead of Security & Compliance center

- Changed something about being able to decide when the first and second notification will come to the group owner, it's not true.

Best regards
/Simon
-

### CODE REVIEW

**Comment by MikePlumleyMSFT** [comment]
@ArunMSFT can you review?

File: microsoft-365/solutions/microsoft-365-groups-expiration-policy.md [hunk 1]
```diff
@@ -23,11 +23,11 @@ description: "Learn about Microsoft 365 groups expiration policies."
...
+When a group expires, almost all of its associated services (the mailbox, Planner, SharePoint site, team, etc.) are also deleted.
 
 When a group expires it is "soft-deleted" which means it can still be recovered for up to 30 days.
 
-Administrators can specify an expiration period and any inactive group that reaches the end of that period, and is not renewed, will be deleted. (This includes archived teams.) The expiration period begins when the group is created, or on the date it was last renewed. Group owners will automatically be sent an email before the expiration that allows them to renew the group for another expiration interval. Teams users will see persistent notifications in Teams.
+Administrators can specify an expiration period and any inactive group that reaches the end of that period, and is not renewed, will be deleted. (This includes archived teams.) The expiration period begins when the group is created, or on the date it was last renewed. Group owners will automatically be sent an notification before the expiration that allows them to renew the group for another expiration interval. Expiration notices for groups used in Teams appear in the Teams Owners feed.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/microsoft-365-groups-expiration-policy.md]:
```suggestion
Administrators can specify an expiration period and any inactive group that reaches the end of that period, and is not renewed, will be deleted. (This includes archived teams.) The expiration period begins when the group is created, or on the date it was last renewed. Group owners will automatically be sent a notification before the expiration that allows them to renew the group for another expiration interval. Expiration notices for groups used in Teams appear in the Teams Owners feed.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @simonagren, kindly commit the suggested change. Many thanks.

**Comment by simonagren** [comment]
Thanks @JohanFreelancer9, looks much better!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Gentle reminder. Thanks

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT It is not merged yet. Please look into it. Thanks

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Kindly check. Thanks

**Comment by yogkumgit** [comment]
@tiburd

**Pull request merged**